modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase

```diff
@@ -1177,6 +1177,54 @@ definition {
 			fieldType = "Text");
 	}
 
+	@description = "LPS-184370 - Verify if is possible delete an custom object that is inactive"
+	@priority = 3
+	test CanDeleteAnInactiveObject {
+		task ("Given a custom object with a text field") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 184370",
+				objectName = "CustomObject184370",
+				pluralLabelName = "Custom Objects 184370");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Text",
+				fieldLabelName = "Text Field",
+				fieldName = "customObjectField",
+				fieldType = "String",
+				isRequired = "false",
+				objectName = "CustomObject184370");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject184370");
+		}
+
+		task ("And given an entry is created") {
+			ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+				fieldName = "customObjectField",
+				objectName = "CustomObject184370",
+				value = "Entry Test");
+		}
```

Comment:
Combine this with the previous step:

`Given a custom object with a text field and an object entry`